project2/ingredients.txt

```diff
@@ -0,0 +1,26 @@
+ADDRESS
```

Comment:
not sure why we want this file in the repo? 

and if we do maybe `sample_recipe.txt` would be a better name?

please remove or rename